src/codegen/program.lisp

```diff
@@ -27,10 +27,13 @@
    (#:rt #:coalton-impl/runtime)
    (#:tc #:coalton-impl/typechecker))
   (:export
+   #:*codegen-hook*
    #:compile-translation-unit))
 
 (in-package #:coalton-impl/codegen/program)
 
+(defvar *codegen-hook*)
```

Comment:
It might be nice to have a docstring or comment for this variable.

Reply:
completely agree, added it, thank you